### PR TITLE
Feat(COCO-STQA-20) Show reminders in calendar

### PR DESCRIPTION
**Pull request by LouisChoiniere**
image

### CODE REVIEW

File: src/main/webapp/resources/javascript/Calendar.js [hunk 1]
```diff
@@ -2,28 +2,34 @@ var appointments;
 var days = $("tbody tr td span.date").toArray();
 var currMonth = getMonthFromString($("#currentMonth").text()).toString();
 var currYear = $("#currentYear").text();
-// var currDay = $("td.current-day").text().trim();
+
+function getMonthFromString(mon) {
+    return new Date(Date.parse(mon + " 1, 2020")).getMonth() + 1
+}
+
```
Comment (Shayminn) [inline src/main/webapp/resources/javascript/Calendar.js:9]: Good refactoring!

File: src/main/java/org/springframework/samples/petclinic/web/EventController.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+@Controller
+public class EventController {
+
+    private final ClinicService clinicService;
+
+    @Autowired
+    public EventController(ClinicService clinicService) {
+        this.clinicService = clinicService;
+    }
+
```
Comment (Shayminn) [inline src/main/java/org/springframework/samples/petclinic/web/EventController.java:22]: This method could also be tested with mockMvc.

**Review by Shayminn — APPROVED** [review]
Left some comments, code reviewed, everything looks good!

File: src/main/webapp/resources/javascript/Calendar.js (hunk 1 above)
Comment (CarbonFanatic) [reply src/main/webapp/resources/javascript/Calendar.js:9]: It is Much cleaner now

**Comment by CarbonFanatic** [comment]
A few more test cases could have been added but overall, the code is very good and works as intended.
All test cases passed.
Only advice would be include more comments in your work. Will make it much easier to follow

**Review by CarbonFanatic — APPROVED** [review]

**Comment by CarbonFanatic** [comment]
[CodeReview-PullRequest-COCO_STQA20.xlsx](https://github.com/cgerard321/STQA-pet-clinic-F2020/files/5682316/CodeReview-PullRequest-COCO_STQA20.xlsx)

**Pull request merged**